docs/getting-started/connect-widget.mdx

```diff
@@ -13,17 +13,207 @@ Here's a preview:
+            }){
+              Image(systemName: "arrowtriangle.right.fill")
+                    .font(.title)
+                    .foregroundColor(.blue)
+                    .padding()
+            }
+          }
+        }
+      }
+}
+
+struct MetriportWidget: UIViewRepresentable {
+    
+    var url: URL
+    private var webView: WKWebView?
+    
+    public init(url: URL) {
+        self.webView = WKWebView()
+        self.url = url
+    }
+    
+    public func makeUIView(context: Context) -> WKWebView {
+        return webView!
+    }
+
+    public func updateUIView(_ webView: WKWebView, context: Context) {
+       let request = URLRequest(url: url)
+       webView.load(request)
+    }
+    
+    public func goBack(){
+        webView?.goBack()
+    }
+
+    public func goForward(){
+        webView?.goForward()
+    }
+    
+    public func refresh() {
+        webView?.reload()
+    }
+}
+```
+
+<Tip>
+  Our GitHub repo has [an example implementation](https://github.com/metriport/metriport/tree/develop/examples/ios/Metriport%20mobile) of this in an iOS app.
+</Tip>
+
+### Android (Kotlin)
+
+<CodeGroup>
+
+```kotlin MainActivity.kt
+
+package com.example.metriport
+
+import android.os.Bundle
+import android.webkit.WebViewClient
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.activity_main.*
+
+
+class MainActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        webView.webViewClient = WebViewClient()
+
+        // For more information of how to create a token view link below
+        // https://docs.metriport.com/getting-started/connect-quickstart#4-link-to-the-metriport-connect-widget-in-your-app
+        val prodUrl = "https://connect.metriport.com/?token=demo"
+
+        // To use emulator locally make sure to use your IP
+        // val localhostUrl = "http://{LOCAL_IP}:3001/?token={YOUR_TOKEN}"
+
+        webView.loadUrl(prodUrl)
+
+        webView.settings.domStorageEnabled = true;
+
+        webView.settings.javaScriptEnabled = true
+
+        webView.settings.setSupportZoom(true)
+    }
+
+    override fun onBackPressed() {
+        if (webView.canGoBack())
+            webView.goBack()
+        else
+            super.onBackPressed()
+    }
+}
+
+```
+
+```xml activity_main.xml
+
+
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".MainActivity">
+
+    <!-- Place Web-view on the Screen -->
+    <WebView
+        android:id="@+id/webView"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"/>
+
+</androidx.constraintlayout.widget.ConstraintLayout>
+
+```
+
```

Comment:
We should probably add the AndroidManifest.xml file 

Or at least mention the user permissions that need to be in there 

    <uses-permission android:name="android.permission.INTERNET"/>
    <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />

Reply:
Good call, will fix